### ISSUE TITLE
Create as an Xposed module

### ISSUE THREAD

**Issue by johanw666** [fengtimozz/Android-CipheringIndicator-API#4]
In the development roadmap I see this as final step: Create a pull request for adding the final API to Android Open Source.

However, should Google and/or phone manufacturers choose not to implement this, there needs to be a fallback. The Xposed framework could be ideal for this, also to bring this functionality to existing phones.

**Comment by SecUpwN** [comment]
Hey @johanw666, I appreciate that you'd like to see development here. But there are two large Issues:
1. The owner of this repository, @PrivacyCollective does not answer emails, nor maintain this repo.
2. Google does not seem to be making ANY effort in resolving [Android Issue 5353](https://code.google.com/p/android/issues/detail?id=5353), nor offering help.

I will try to get back in touch with the owner of this repo once more, please closely monitor this and stay alert, once activity comes to this repo. I secretly hope that @PrivacyCollective will come back soon.

**Comment by johanw666** [comment]
I noticed that, and I saw you forked it. But I can't open issues on your fork.

My proposal was exectly aimed at working around Google not being supportive, as Xposed is used for changing Android at boot.

**Comment by SecUpwN** [comment]
> I noticed that, and I saw you forked it. But I can't open issues on your fork.

Yes, because it does make no sense. I have requested `COLLABORATOR` status though, which would mean I could maintain this project as well. Until then, you should hang in tight until I received a reply. Thanks!
